### DISCUSSION TITLE
Why the default configuration show me always python and the cloud icons?

### DISCUSSION THREAD

**Discussion by toti1212** [starship/starship#2181] [Q&A] [answered]
I just installed Starship and once I modify my (very default) .zshrc file adding the eval command, I always see this as my default prompt:

![Screen Shot 2021-01-21 at 12 07 57](https://user-images.githubusercontent.com/8757113/105369441-52cf0400-5be1-11eb-90f3-f0d86fd09448.png)

It happens in any folder (Documents, Desktop, etc), except those that are a git project.
How can I remove this behavior?

**Comment by davidkna** [comment, marked as answer]
I think you have a `*.py` file in each of those folders. You can disable `*.py` file scanning by adding this to your starship config:

```toml
# ~/.config/starship.toml
[python]
scan_for_pyfiles = false

```

**Reply by davidkna** [reply]
Forgot about the cloud icon:

I think the cloud icon is the AWS module showing your current AWS region (you can check which module it is with `starhip explain`)

To hide it unset `AWS_REGION`/ `AWS_DEFAULT_REGION` or disable it with:

```toml
[aws]
disabled = true
```

**Comment by toti1212** [comment]
Yay! thanks for the reply! 🙌  With these suggestions I have it working perfectly! Indeed, I had .py files everywhere, and I also had the .aws folder configured.
